cpu/msp430/mtarch.c

```diff
@@ -66,10 +66,10 @@ mtarch_start(struct mtarch_thread *t,
 
   t->sp = &t->stack[MTARCH_STACKSIZE - 1];
 
-  *t->sp = (unsigned short)mt_exit;
+  *t->sp = (unsigned short)(void *)mt_exit;
```

Comment:
Question : is it allowed on 20bits version of msp ? I believe far pointers are to be stored in 32 bits integers.